### PR TITLE
task1

### PR DESCRIPTION
**Pull request by HugoGv**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      madZombies = document.querySelector('.mad-zombies'),
+      strongZombies = document.querySelector('.strong-zombies');
+
+let totalZombiesCount = 0,
+    deadZombiesCount = 0,
+    smallZombiesCount = 0,
+    madZombiesCount = 0,
+    strongZombiesCount = 0;
+
+countZombieType = (zombies) => {
```
Comment (ExSamaelSolbeg) [inline js/app.js]: is there a chance to implement this task without any function?

File: js/app.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+const totalZombies = document.querySelector('.total-zombies'),
+      deadZombies = document.querySelector('.dead-zombies'),
+      smallZombies = document.querySelector('.small-zombies'),
+      madZombies = document.querySelector('.mad-zombies'),
+      strongZombies = document.querySelector('.strong-zombies');
```
Comment (ExSamaelSolbeg) [inline js/app.js]: please define a keyword for each variable
```
const totalZombies;
const deadZombies;
...
```

File: js/app.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+let totalZombiesCount = 0,
```
Comment (ExSamaelSolbeg) [inline js/app.js]: same here

File: js/app.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    deadZombiesCount = 0,
+    smallZombiesCount = 0,
+    madZombiesCount = 0,
+    strongZombiesCount = 0;
+
+countZombieType = (zombies) => {
+    for (let i = 0; i < zombies.length; i++) {
+        let zombie = zombies[i];
+
+        let isNumber = typeof zombie === 'number';
```
Comment (ExSamaelSolbeg) [inline js/app.js]: please implement this task without "typeof" keyword

File: js/app.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            strongZombiesCount++;
+        }
+
+        totalZombiesCount++;
+    }
+};
+
+countZombieType(zombiesData);
+
+totalZombies.textContent = totalZombies.textContent + '{' + totalZombiesCount + '}';
```
Comment (ExSamaelSolbeg) [inline js/app.js]: 1) missing console.log command
2) use text formatting instead of '{' + totalZombiesCount + '}';

File: js/app.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+let totalZombiesCount = 0,
+    deadZombiesCount = 0,
+    smallZombiesCount = 0,
+    madZombiesCount = 0,
+    strongZombiesCount = 0;
+
+countZombieType = (zombies) => {
+    for (let i = 0; i < zombies.length; i++) {
+        let zombie = zombies[i];
```
Comment (ExSamaelSolbeg) [inline js/app.js]: const

File: js/app.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+        let isNumber = typeof zombie === 'number';
+
+        let isDeadZombie = zombie > 0 && zombie < 11;
```
Comment (ExSamaelSolbeg) [inline js/app.js]: const
const
const

**Review by ExSamaelSolbeg — CHANGES_REQUESTED** [review]

File: js/app.js (hunk 1 above)
Comment (HugoGv) [reply js/app.js]: Yes, of course.

File: js/app.js (hunk 2 above)
Comment (HugoGv) [reply js/app.js]: fixed

File: js/app.js (hunk 3 above)
Comment (HugoGv) [reply js/app.js]: fixed

File: js/app.js (hunk 6 above)
Comment (HugoGv) [reply js/app.js]: fixed

File: js/app.js (hunk 7 above)
Comment (HugoGv) [reply js/app.js]: fixed

**Review by ExSamaelSolbeg — DISMISSED** [review]
mostly ok
it would be nice to implement string formatting

**Comment by HugoGv** [comment]
> mostly ok
> it would be nice to implement string formatting

string formatting added

**Review by ExSamaelSolbeg — APPROVED** [review]

**Pull request merged**